src/source.c

```diff
@@ -1532,8 +1532,13 @@ struct dispatch_timer_s _dispatch_timer[] =  {
 #define DISPATCH_TIMER_COUNT \
 		((sizeof(_dispatch_timer) / sizeof(_dispatch_timer[0])))
 
+#if __linux__
```

Comment:
On Linux we only have kevent, not kevent64 (os/linux_base.h #defines kevent64 to be kevent).  The type of the udata field of kevent is void* (vs. kevent64 where the type of the udata field is uintptr_t).     It would be nicer to use a typeof here to avoid a #if __linux__ , but I didn't see a good way to do that through all the levels of macros being defined.